### ISSUE TITLE
Max button should set value of fan to 7

### ISSUE THREAD

**Issue by tgsoverly** [tgsoverly/smart-things#2]

**Comment by jterrace** [comment]
Why is 7 used as the max? Mine goes up to 10

**Comment by tgsoverly** [comment]
The version I built this off of went to 7.  🤷‍♂️

**Comment by jterrace** [comment]
Interesting. What model number and software version do you have? Mine is 3200 WHF and software version 2.15.1.

**Comment by tgsoverly** [comment]
I actually don't own one, I did this off a friends one.  Do you have the
fan with the new controller with all the blue lights?

**Comment by jterrace** [comment]
No, I bought the older generation 2 controller instead of using the new capacitive fancy one (from here https://blog.airscapefans.com/archives/new-controls). You can actually see on that page in the last paragraph, they say it's not compatible at all with the remote control or API. I'm not sure why they went with a capacitive switch anyway, since capacitive switches are awful for usability.

I can make the max value a parameter so people can pick. Unfortunately, I don't see "10" anywhere in the API responses to be able to set it programmatically. Though it's possible it might depend on the software version, which is available.

Anyway, I have a lot of improvements I could send you, but I'm not allowed to contribute to an open source project without a license by policy of where I work. So if you add a license, I'll send you my changes.

**Comment by tgsoverly** [comment]
As I remember (from two years ago) the commands were only up and down so
you would have to step up the correct number.  I have added a license if
you want to throw up a PR.
